examples/download.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The vmnet-helper authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Download disk image for the examples.
+
+DISK_IMAGE="${1:?Usage: disk.sh FILENAME}"
+TMP_IMAGE="$DISK_IMAGE.tmp"
+
+trap 'rm -f "$DISK_IMAGE.tmp"' EXIT
+
+# Download qcow2 cloud image.
+curl \
+    --fail \
+    --location \
+    --output "$TMP_IMAGE" \
+    https://cloud-images.ubuntu.com/releases/24.10/release/ubuntu-24.10-server-cloudimg-arm64.img
```

Comment:
Maybe indicate clearly in a comment at the beginning that the script needs changes for amd64 support? Alternatively, `$(arch)` could help to support both, but it apparently returns `x86_64` on intel, and the ubuntu URL uses `amd64`.

Reply:
I'll add a comment about using amd64 image on intel. I don't want to maintain the logic for multiple architecture in these bash scripts. This is already done in the python example.

Reply:
Yeah definitely not worth the complication.

Reply:
Will use 

```shell
"https://cloud-images.ubuntu.com/releases/24.10/release/ubuntu-24.10-server-cloudimg-$IMAGE_ARCH.img"
```